app/src/main/java/io/gnosis/safe/ui/safe/overview/SafeOverviewViewModel.kt

```diff
@@ -0,0 +1,39 @@
+package io.gnosis.safe.ui.safe.overview
+
+import androidx.lifecycle.ViewModel
+import pm.gnosis.svalinn.common.PreferencesManager
+import pm.gnosis.svalinn.common.utils.edit
+import javax.inject.Inject
+
+class SafeOverviewViewModel
```

Comment:
I know is just naming, but I would rename this to `TermsViewModel` and put under the package `io.gnosis.safe.ui.terms` as these don't have to be coupled with the overview screen necessarily.

Reply:
Of course along with the bottom sheet fragment. Similarly to what we have for the EnsInputDialog.

Reply:
As soon as PR#580 is merged, I will have to change this.